src/app/loadout/LoadoutPopup.tsx

```diff
@@ -97,6 +98,7 @@ class LoadoutPopup extends React.Component<Props> {
 
   render() {
     const { dimStore, previousLoadout, loadouts, query, onClick } = this.props;
+    const sortedLoadouts = _.sortBy(loadouts, ['classType', 'name']);
```

Comment:
We prefer not to use the string shorthand form of lodash methods. We have a composable comparator pattern that we use instead, in `comparators.ts`.

Reply:
Class methods tend to be arrow functions so that they capture the proper this reference. For standalone functions we always use the function syntax.

Reply:
Cool, did a bit of googling and see what you guys are doing now.